portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,18 +14,34 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Arrays;
 
-/** Servlet that returns some example content. */
+/** Servlet that return some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+
+  ArrayList<String> msglist = new ArrayList<String> (Arrays.asList("Hi!","Ni Hao!","Bonjour!","Hola!"));
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Xin!");
+    response.setContentType("application/json;");
+    response.getWriter().println(this.convertArrayToJsonUsingGson(msglist));
```

Comment:
Question: Is this needed to call convertArrayToJsonUsingGson()?

Reply:
They did wrap it in the sample code, which I also don't think is quite needed. I can directly convert this to a line of code